README.md

```diff
@@ -47,17 +46,11 @@ This is a react app that allows users to make simple calculations.
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
@HAMMAS-SALEEM noted, thanks for the thorough review and help, I'll definitely update the readme file